AGGREGATE.md

```diff
@@ -187,6 +187,26 @@ The `payload` will need to contain all the information needed for the aggregatio
 
 The payload should be encrypted via [HPKE](https://datatracker.ietf.org/doc/draft-irtf-cfrg-hpke/), to public keys specified by the processing origins at some well-known address  `/.well-known/aggregation-service/keys.json` that the browser can fetch. Note that we are avoiding using the `attribution-reporting` namespace because many APIs may want to use this infrastructure beyond attribution reporting.
 
+The  processing origin will respond with a set of keys and a cache control header which dictates how long to store the keys for. The browser will use these keys for the max-age as specified by the header. The scheme of the JSON encoded public keys is as follows:
```

Comment:
optional: s/and a cache control header which dictates/which will be stored according to standard HTTP caching rules, i.e. using Cache-Control headers to dictate/
optional: s/max-age/freshness lifetime/

Do we want to mention that the browser might want to enforce minimum/maximum freshness lifetimes?

Reply:
Updated based on suggestions. Thanks both.